tests/models/test_owlv2.py

```diff
@@ -57,3 +57,22 @@ def test_successful_image_detection_with_nms():
 
     for pred in results[0]:
         assert pred.label in prompts
+
+
+def test_successful_image_with_large_prompt():
+    test_image = "000000039769.jpg"
+    prompts = [
+        """a photo of a  cat that is spleeping next to a remote control,
+            the cat has a light brown color with black spots and seems to be wearing a light green necklace.
+            It also seems to be stretching its right leg and next to its left leg, it is stepping on the tail"""
```

Comment:
Using `"""` will actually add multiple spaces to the prompt, it is ok I guess for testing long texts, but ideally the prompt should be separated by single space.

Reply:
Yes, for testing purposes it does not really matter much.